app/Models/Feed.php

```diff
@@ -287,7 +287,7 @@ public function load($loadDetails = false, $noCache = false) {
 				if ((!$mtime) || $simplePie->error()) {
 					$errorMessage = $simplePie->error();
 					throw new FreshRSS_Feed_Exception(
-						($errorMessage == '' ? 'Unknown error for feed' : $errorMessage) . ' [' . $url . ']'
+						($errorMessage == '' ? 'Unknown error for feed' : $errorMessage) . ' [' . $this->url . ']'
```

Comment:
`$this->url` does not contain the `user:password` component, unlike `$url` which does